CMakeLists.txt

```diff
@@ -14,6 +14,19 @@ endforeach()
 
 project(CudaCommon)
 
+## CudaCommon Version
+set(CUDACOMMON_VERSION_MAJOR "2")
```

Comment:
Should these variables be CUDACOMMON_VERSION_* or CudaCommon_VERSION_* as all other variables in the exported `CudaCommon.cmake` (`CudaCommon_INCLUDE_DIRS`, `CudaCommon_LIBRARIES`, etc.)?

Reply:
@LucasGandel

Reply:
Yes CudaCommon_VERSION is better probably